capabilities/grafana/validate.yaml

```diff
@@ -3,5 +3,8 @@ tasks:
     actions:
       - description: Validate...
         cmd: "echo Replace Me"
+
         # wait:
-        #   cluster:
+        #   network:
```

Comment:
for now we've just been adding a healthcheck to some k8s resource, we can do the network tests in another PR

Reply:
Do you want me to add the resource checks in the validate for each of the modified apps now? I can do that.

Reply:
threw in some checks on the pods. maybe should have used the deployments, stateful sets, etc instead. Assuming it makes sense can flesh out more/better checks in the future.